apps/server/src/modules/system/controller/mapper/system-oauth-response.mapper.spec.ts

```diff
@@ -38,6 +38,7 @@ describe('oauth-response mapper', () => {
 		issuer: 'mock_issuer',
 		jwksEndpoint: 'mock_jwksEndpoint',
 		redirectUri: 'mock_codeRedirectUri',
+		provisioningUrl: 'mock_provisioning_url',
```

Comment:
Is this neccessary ?

Reply:
not anymore as it is not a part of the oauthconfig anymore